api/configs/feature/__init__.py

```diff
@@ -136,6 +136,12 @@ class EndpointConfig(BaseSettings):
         default="",
     )
 
+    PROXY_FIX_MIDDLEWARE_ENABLED: bool = Field(
```

Comment:
```suggestion
    RESPECT_XFORWARD_HEADERS_ENABLED: bool = Field(
```
As `ProxyFix` is not a common concept and it's a alias for `X-Forwarded-For Proxy Fix` inside `werkzeug` library, it's suggested to change to a non-implementation-related config name.

Reply:
Renamed.